rfcs/framework-0063-notification-system.md

```diff
@@ -0,0 +1,422 @@
+- Start Date: 2022-06-13
+- RFC PR: [#63](https://github.com/inveniosoftware/rfcs/pull/63)
+- Authors: David Eckhard, Zacharias Zacharodimos, Alex Ioannidis
+
+# Notification System
+
+## Summary
+
+Implementation details for a notification system to be used in InvenioRDM.
+
+## Motivation
+
+As the product is getting larger and new features are added, it is hard to keep an overview of everything going on as a user. With the communities feature, certain tasks require an action to be executed by the user (f.e. accepting an incoming request for a community). This incoming request can easily get "lost", if a user does not explicitly check the community dashboard. Therefore, a notification system will not only solve this problem (and similar ones) but also increase user engagement by advertising the repository some more.
+
+It will take care of sending the notifications based on the configuration and user preferences (i.e. if the repository disables certain notifications or a user does not want to receive certain notifications, they should not be sent).
+
+### Use Case
+
+- Send notifications for operations involving multiple users (see [table](#notification-examples))
```

Comment:
Many points of the enumeration are hard to grasp and the implications are mixed (e.g. at a user profile level, at a template or instance level, etc.). Maybe using the templated [as a ... I want ... so that ...](https://github.com/inveniosoftware/rfcs/blob/master/0000-template.md#motivation) structure would clarify the use cases

Reply:
plus one to this :)